src/MainMenu.cpp

```diff
@@ -1769,4 +1769,256 @@ ZunResult MainMenu::ChoosePracticeLevel()
+        {
+            this->setSavedCursorPosition(&this->vm[i + 0x43], i, i, this->cursor);
+        }
+        else
+        {
+            this->setSavedCursorPosition(optionsVm, i, i, this->cursor);
+            optionsVm++;
+        }
+    }
+
+    for (i = 0; i < 5; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_LIFECOUNT, i, g_Supervisor.cfg.lifeCount);
+    }
+
+    for (i = 0; i < 4; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_BOMBCOUNT, i, g_Supervisor.cfg.bombCount);
+    }
+    for (i = 0; i < 2; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_COLORMODE, i, g_Supervisor.cfg.colorMode16bit);
+    }
+    for (i = 0; i < 2; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_PLAYSOUNDS, i, g_Supervisor.cfg.playSounds);
+    }
+    optionsVm = &this->vm[0x4d];
+
+    for (i = 0; i < 3; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_MUSICMODE, i, g_Supervisor.cfg.musicMode);
+    }
+    optionsVm = &this->vm[0x4b];
+    for (i = 0; i < 2; i++, optionsVm++)
+    {
+        this->setSavedCursorPosition(optionsVm, CURSOR_OPTIONS_POS_SCREENMODE, i, this->windowed);
+    }
+    if (this->stateTimer >= 0x20)
+    {
+        if (WAS_PRESSED_WEIRD(TH_BUTTON_LEFT))
+        {
+            switch (this->cursor)
+            {
+            case CURSOR_OPTIONS_POS_LIFECOUNT:
+
+                g_SoundPlayer.PlaySoundByIdx(SOUND_MOVE_MENU, 0);
+                if (g_Supervisor.cfg.lifeCount <= 0)
+                {
+                    g_Supervisor.cfg.lifeCount = 5;
+                }
+                g_Supervisor.cfg.lifeCount = g_Supervisor.cfg.lifeCount - 1;
+                break;
+
+            case CURSOR_OPTIONS_POS_BOMBCOUNT:
+
+                g_SoundPlayer.PlaySoundByIdx(SOUND_MOVE_MENU, 0);
+                if (g_Supervisor.cfg.bombCount <= 0)
+                {
+                    g_Supervisor.cfg.bombCount = 4;
+                }
+                g_Supervisor.cfg.bombCount = g_Supervisor.cfg.bombCount - 1;
+                break;
+
+            case CURSOR_OPTIONS_POS_COLORMODE:
+
+                g_SoundPlayer.PlaySoundByIdx(SOUND_MOVE_MENU, 0);
+                if (g_Supervisor.cfg.colorMode16bit <= 0)
+                {
+                    g_Supervisor.cfg.colorMode16bit = 2;
+                }
+                g_Supervisor.cfg.colorMode16bit = g_Supervisor.cfg.colorMode16bit - 1;
+                break;
+
+            case CURSOR_OPTIONS_POS_MUSICMODE:
+
+                g_SoundPlayer.PlaySoundByIdx(SOUND_MOVE_MENU, 0);
+                g_Supervisor.StopAudio();
+                if (g_Supervisor.cfg.musicMode <= OFF)
+                {
+                    g_Supervisor.cfg.musicMode = MIDI + 1;
+                }
+                g_Supervisor.cfg.musicMode = g_Supervisor.cfg.musicMode - 1;
```

Comment:
```suggestion
                g_Supervisor.cfg.musicMode -= 1;
```

While you can't use --, you can use -= I believe.